internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1171,7 +1171,12 @@ func UncordonNode(seq runtime.Sequence, data interface{}) (runtime.TaskExecution
 
 		var kubeHelper *kubernetes.Client
 
-		if kubeHelper, err = kubernetes.NewClientFromKubeletKubeconfig(); err != nil {
+		if err = retry.Constant(5*time.Minute, retry.WithUnits(time.Second), retry.WithErrorLogging(true)).RetryWithContext(ctx,
```

Comment:
etcd recovery triggered a bug here: kubelet cert might not be ready by the time boot sequence comes here